ansible/roles/debops.sftpusers/tasks/sites.yml

```diff
@@ -20,7 +20,7 @@
     state: 'directory'
     owner: 'root'
     group: '{{ item.group | default(item.name + sftpusers_name_suffix) }}'
-    mode: '2775'
```

Comment:
Please double check the mode parameter in this file.

Reply:
The value with leading zero works fine with the `file` Ansible module.